repl/src/main/scala/com/cloudera/livy/repl/ReplDriver.scala

```diff
@@ -94,20 +94,23 @@ class ReplDriver(conf: SparkConf, livyConf: RSCConf)
   }
 
   override protected def createWrapper(msg: BaseProtocol.BypassJobRequest): BypassJobWrapper = {
-    interpreter match {
-      case pi: PythonInterpreter => pi.createWrapper(this, msg)
```

Comment:
If Job is submitted before interpreter is fully launched, here `interpreter` will be null. This will offload `createWrapper` to `super.xxx`, and it will use kryo to deserialize pickle serialized job, this will lead to kryo exception.